Makefile

```diff
@@ -40,6 +40,7 @@ clean:
 	rm -rf docs/_build
 
 install:
+	python -m pip install poetry
```

Comment:
The documentation of Poetry doesn't recommend installing it with pip because Poetry's dependencies will also be installed, which might cause conflicts with other packages (https://python-poetry.org/docs/#alternative-installation-methods-not-recommended).

I think the best solution here is to either skip it and let users decide how to install Poetry (that's what we do now but our documentation should definitely provide some guidance on how to do that) or move it to a separate step inside a Makefile.

Reply:
Ah that's new to me. Fair enough. Will close this one for now then.